app/src/org/commcare/dalvik/application/CommCareApplication.java

```diff
@@ -967,20 +974,6 @@ public CommCareSessionService getSession() throws SessionUnavailableException {
 
     ArrayList<NotificationMessage> pendingMessages = new ArrayList<NotificationMessage>();
 
-    Handler toaster = new Handler() {
-        /*
-         * (non-Javadoc)
-         * @see android.os.Handler#handleMessage(android.os.Message)
-         */
-        @Override
-        public void handleMessage(Message m) {
-            NotificationMessage message = m.getData().getParcelable("message");
-            Toast.makeText(CommCareApplication.this,
-                    Localization.get("notification.for.details.wrapper", new String[]{message.getTitle()}),
-                    Toast.LENGTH_LONG).show();

```

Comment:
This was getting a memory leak lint warning due to the fact that an inner class maintains a reference to its parent class, hence if the Handler object remains around (which happens until messages linked to it are processed) and the context is no longer used, the context will not be garbage collected. 

I know Application objects live for the span of the app, so this is sort of pointless, but I figured I'd still refactor it.